apps/docs/content/components/Card.mdx

```diff
@@ -17,7 +17,7 @@ import {Card} from '@primer/react-brand'
 
 ### Default
 
-The default card can be built with a custom heading and a description. The link label defaults to `Read more` and can be customized using the `link` prop.
+The default card displays a heading and description. The link text defaults to `Read more`, but can be customized using the `label` prop.
```

Comment:
General comment about the docs: can we put it in correct alphabetical order in the nav.yml please @josepmartins? Forgot to mention this in previous PR.

![Card appearing between Avatar and AnchorNav](https://user-images.githubusercontent.com/13340707/224055310-55d85a98-c823-49c2-a100-df6575c185e9.png)

Reply:
Done!